server/helpers/createSocialUsers.js

```diff
@@ -0,0 +1,50 @@
+import models from '../database/models';
```

Comment:
Please, do you think it's right to create users and sessions in a helper file?

Reply:
the reason for abstracting them away in the helper is to prevent my controller class from having duplication of similar blocks of codes, as other passport strategies might implement same session and user creation multiple times.